### DISCUSSION TITLE
save a tweet thread

### DISCUSSION THREAD

**Discussion by s-kris** [plhery/node-twitter-api-v2#119] [Ideas]
It'll be great to have a function to save a tweet thread. 

Thank you! :)

**Comment by alkihis** [comment]
Hi,
It's not as easy as you think:
If you have the "latest" tweet of a thread, you can crawl it through a loop of `in_reply_to_id_str`, but on the otherside, you need to make a tweet search. That is limited to 7-days search on both v1 and v2 basic API. So it might be confusing to include a function that can crawl a thread... but only using the last tweet.

**Reply by s-kris** [reply]
Ohh, yes. Thank you!
